shared_model/interfaces/iroha_internal/transaction_batch_parser.hpp

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_TRANSACTION_BATCH_PARSER_HPP
+#define IROHA_TRANSACTION_BATCH_PARSER_HPP
+
+#include "interfaces/common_objects/range_types.hpp"
+#include "interfaces/common_objects/transaction_sequence_common.hpp"
+
+namespace shared_model {
+  namespace interface {
+
+    /**
+     * Parses a transaction list and return a list of possible batches
+     */
+    class TransactionBatchParser {
+     public:
+      virtual std::vector<types::TransactionsForwardCollectionType>
```

Comment:
Why have you decided not to put any docs about hose methods?

Reply:
They all match the description of the class, and argument types describe the differences.